docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -559,9 +573,10 @@ The `reload` method returns a `boolean` indicating whether the reload was succes
 A value of `true` means the reload operation was successful, not necessarily that there were updates to the certificates.
 
 After a `TlsConfiguration` has been reloaded, servers and clients using this configuration may need to perform specific actions to apply the new certificates.
-The recommended approach is to fire a CDI event (`CertificateReloadedEvent`) that servers and clients can listen to and make the necessary changes:
 
-[source, java]
+* The recommended approach is to trigger a CDI event,`CertificateReloadedEvent` (this is a custom class you have to implement, it’s not provided by {project-name}), for servers and clients to listen to and update their configurations as shown below:
```

Comment:
```suggestion
* The recommended approach is to trigger the provided CDI event, `CertificateReloadedEvent`, a custom class available in Quarkus, which servers and clients can listen to in order to update their configurations, as shown below:
```

Reply:
@cescoffier Hello Clement! Do you agree with this new formulation above, which is based on your feedback?